sample_application/config.py

```diff
@@ -25,3 +25,4 @@
 SAMPLE_APPLICATION_CLIENT = Client(
     {'TOKEN': SAMPLE_APPLICATION_ADSWS_API_TOKEN}
 )
+SQLALCHEMY_BINDS = {'sample':'sqlite:///:memory:'}
```

Comment:
In principle, the maintainer of a microservice doesn't need to manage `SQLALCHEMY_BINDS` unless their microservice uses multiple databases: using `SQLALCHEMY_URI` is fine. 

We internally need to manage `SQLALCHEMY_BINDS` if and only if we host multiple applications in the same container. In effect, the only *necessary* thing for the developer to have is the  `__bind_key__` to their models.

I'd suggest removing `SQLALCHEMY_BINDS` from the repo for these reasons, and maybe adding a short comment to `__bind_key__` just to note that we do expect this to be defined.